content/_index.md

```diff
@@ -12,6 +13,7 @@ show_featured_story: false
 show_featured_footer: false
 layout: single
 header_wrapper_class: "header-default-bg-img"
```

Comment:
How about you create a new class for header_wrapper_class: "header-default-bg-img header-custom-sam-iot".

This will allow you to scope your CSS change to this new class:

header-custom-sam-iot .jumbotron h2 {
  font-size: 27px;
}